cmake/Modules/CMakeFindFrameworks.cmake

```diff
@@ -0,0 +1,32 @@
+# Distributed under the OSI-approved BSD 3-Clause License.  See accompanying
```

Comment:
here and other files in `Modules` dir: - this is _a lot_ of third-party-code and i would rather not include anything but absolutely essential code into project to reduce future maintenance cost. Could you please argue why it is needed?

Also, could you please rename to `modules` (no capitalization) so it in line with rest of directory structures

Reply:
Content:
These files are just equivalents of m4 files for autotools + some macros.
Most are from the cmake.
The typical structure of cmake project contains the 'cmake' directory with 
cmake/Modules   <- here one has files that search for dependencies. Typically needed to search for SW that is not very common and has no cmake configs.
cmake/Templates  <-  configs to be installed for this project

Name: 
That comes as a standard from cmake as well.  
For instance I have '/usr/share/cmake/Modules' with capital M.